WordPress/gutenberg
# [RNMobile] Remove ReactAztecTextInputShadowNodeFork

## Description
The changes that required the `ReactTextInputShadowNodeFork` class are now included in the`ReactTextInputShadowNode` class since we've updated to React Native 0.64, so this PR is just removing the no longer needed "Fork" class. There should be no behavior changes in the app.

Additional context for why this code was originally introduced can be found [here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1560).

## How has this been tested?
1. Run the demo app
2. Make sure that there has not been a regression in the padding around RichText inputs (see https://github.com/wordpress-mobile/gutenberg-mobile/issues/992)

## Types of changes
This is a refactoring that should result in no behavior changes.

## Checklist:
